docs/Getting_Involved/Testing.rst

```diff
@@ -9,12 +9,26 @@ Executing our Tests
 *coala* has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
-To run tests, you first need to install some dependencies. This can be
-done by executing:
+To run tests, you first need to install some dependencies.
+This can be done by following these steps:
+
+If you have not already, You can clone the
```

Comment:
two things: one) You shouldnt be with a capital Y
two) when you give directions, or steps, it should be in the imperivate mood. So:
``If you have not already, clone the..``